zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/shoot/handler/DamageShotHandler.java

```diff
@@ -40,6 +43,7 @@ public DamageShotHandler(@NotNull Data data) {
      *
      * @return A {@link ConfigProcessor} for {@link Data}s
      */
+    @ProcessorMethod
     public static @NotNull ConfigProcessor<Data> processor() {
```

Comment:
not sure how i feel about keeping ConfigProcessors around just for the sake of data validation